templates/modelExpt/run_modelComp_expt.m

```diff
@@ -0,0 +1,165 @@
+if ~exist(expt.dataPath,'dir')
+    mkdir(expt.dataPath)
+end
+
+if ~isfield(expt,'gender'), expt.gender = get_height; end       % vocal tract length or gender correlate for LPC Order
+
+% In other experiments, instead of assuming LPC, we collect some
+%   speech samples and allow the experimenter to evaluate different LPCs on
+%   that dataset. For simplicity here, LPC is just set arbitrarily.
+if strcmp(expt.gender, 'male')
+    expt.audapterParams.nLPC = 17;
+else
+    expt.audapterParams.nLPC = 15;
+end
+expt.audapterParams.bShift = 1;
+expt.audapterParams.bRatioShift = 0;
+expt.audapterParams.bMelShift = 1;
+expt.audapterParams.fb = 3;          % set feedback mode to 3: speech + noise
+expt.audapterParams.fb3Gain = 0.02;  % gain for noise waveform
+
+%perturbation amount, in mels
+shiftMag = 125;
+
+% timing
+expt.timing.stimdur = 1.9;          % time stim is on screen (in s)
+expt.timing.interstimdur = .75;     % minimum time between stims (in s)
+expt.timing.interstimjitter = .75;  % maximum extra time between stims (in s), ie, jitter
+expt.timing.visualfbdur = 0.75;     % how long to show "speak louder" prompt (in s)
+
+%set up duration feedback parameters
+expt.durcalc.min_dur = .4;          % minimum allowable vowel duration (in s)
+expt.durcalc.max_dur = .65;         % maximum allowable vowel duration (in s)
+expt.durcalc.ons_thresh = 0.15;     % percentage of maximum amplitude for determining onset threshold
+expt.durcalc.offs_thresh = 0.4;     % percentage of maximum amplitude for determining offset threshold
+
+%% Set up duration practice
+exptDur = expt;
+exptDur.session = 'dur'; 
+exptDur.dataPath = fullfile(expt.dataPath,exptDur.session);
+if ~exist(exptDur.dataPath, 'dir')
+    mkdir(exptDur.dataPath)
+end
+exptDur.ntrials = 6;
+
+exptDur.shiftMags   = zeros(1,exptDur.ntrials);
+exptDur.shiftAngles = zeros(1,exptDur.ntrials);
+
+exptDur.words = {'head' 'dead' 'Ted'};
+exptDur.allWords = randomize_wordOrder(length(exptDur.words), exptDur.ntrials);
+exptDur.listWords = exptDur.words(exptDur.allWords);
+
+exptDur.conds = {'noShift'};
+exptDur.allConds = ones(1, exptDur.ntrials);
+exptDur.listConds = exptDur.conds(exptDur.allConds);
+
+% set up duration feedback parameters
+exptDur.bDurFB = ones(1,exptDur.ntrials); % 1 x ntrials vector
+
+exptDur = set_exptDefaults(exptDur);
+
+exptfile = fullfile(exptDur.dataPath,'expt.mat');
+dur.expt = exptDur;
+save(exptfile, '-struct', 'dur')
+fprintf('Saved exptDur file: %s.\n',exptfile);
+
+%% Set up compensation experiment
+% stimuli and vowel list
+expt.words = {'head' 'dead' 'Ted'}; 
+nwords = length(expt.words);
+[vowels,~,ivowels] = unique(txt2arpabet(expt.words));
+if length(vowels) == length(ivowels), vowels = vowels(ivowels); end
+expt = set_missingField(expt,'vowels',vowels);
+
+expt.shiftDirs = {0 -1 1};
+expt.shiftNames = {'noShift' 'shiftDown' 'shiftUp'};
+expt.conds =      {'noShift' 'shiftDown' 'shiftUp'};
+expt.shiftMag = 125;
```

Comment:
Add:
```
expt.shifts.mels{1} = [-shiftMag 0]; % shiftDown (only shift F1)
expt.shifts.mels{2} = [shiftMag 0]; % shiftUp (only shift F1)
```
Also, we define "shiftMag" earlier, so we don't want to hardcode 125 again. I would either set expt.shiftMag = shiftMag, or remove the use of that shiftMag variable and always use expt.shiftMag from the start.